### PR TITLE
DCI Job: respect force-check

### PR DESCRIPTION
**Pull request by fredericlepied**
Test-Hints: sno
Test-Hints: force-check

### CODE REVIEW

**Review by ramperher — APPROVED** [review]
It looks like it's behaving as expected, from what I can see in the PR check

File: .github/workflows/pr.yml [hunk 1]
```diff
@@ -123,6 +123,8 @@ jobs:
             grep -v "issue(s) which need to be resolved\|See error output above for details.\|Command \"ansible-doc -t module .*\" returned exit status .*\." |
             sed -r 's/\x1B\[[0-9]{1,2}[mGK]//g' > "$key.errors"
           done
+          # remove line numbers
+          sed -i -E -e 's/:[0-9]+:/:/' -e 's/:[0-9]+:/:/' branch.errors main.errors
```
Comment (tonyskapunk) [inline .github/workflows/pr.yml:127]: Is the 2nd expresion needed or bad copy/paste?

File: hack/maybe-dci-check-change [hunk 2]
```diff
@@ -36,10 +36,17 @@ proj=$(sed -e 's@https://github.com/\(.*\)/pull/.*@\1@i' <<< $URL)
...
+    force_check=
+fi
+
+# Check if there is a force-check or a code change
+if [ -n "$force_check" ] || curl -s -L "${HEADERS[@]}" "https://api.github.com/repos/$proj/pulls/$pr/files"|jq -r .[].filename|grep -v '\.md$'|grep -E 'roles/|plugins/'; then
     curl -s -L "${HEADERS[@]}" -X POST -d "{\"state\":\"pending\",\"context\":\"$GITHUB_JOBNAME\",\"description\": \"QUEUED\"}" "$statuses_url"
-    dci-check-change --silent "$URL"
+    dci-check-change "$URL"
 else
     curl -s -L "${HEADERS[@]}" -X POST -d "{\"state\":\"success\",\"context\":\"$GITHUB_JOBNAME\",\"description\": \"non code files\"}" "$statuses_url"
```
Comment (tonyskapunk) [inline hack/maybe-dci-check-change:51]: Can we use the same description as in the PR? `"No code change"`

Comment (fredericlepied) [reply hack/maybe-dci-check-change]: right and also to be the same as in dci-merge.sh.

**Review by tonyskapunk — APPROVED** [review]
LGTM

**Pull request merged**